config.py

```diff
@@ -30,6 +30,7 @@ class EnvVars:
             authentication
         gh_token (str | None): GitHub personal access token (PAT) for API authentication
         ghe (str): The GitHub Enterprise URL to use for authentication
+        hide_assignee (bool): If true, the assignee's information is hidden in the output
```

Comment:
The hide_assignee configuration needs to be documented in the README.md file just like hide_author.